python/dwave_qbsolv/qbsolv_binding.pyx

```diff
@@ -63,7 +63,7 @@ def run_qbsolv(Q, num_repeats=50, seed=17932241798878,  verbosity=-1,
 
         def dimod_callback(Q, best_state):
             result = solver.sample_qubo(Q, **sample_kwargs)
-            sample = next(result.samples())
+            sample = result.samples()[0]
```

Comment:
Does this work in all dimod versions?

Reply:
Nope, not with 0.8.1... will remove this change, thanks!

Reply:
Or, we can bump the lower bound for dimod to 0.9?

Reply:
I'm okay with that, but am not sure if such a small change warrants it. I'm assuming something like this has probably happened before..what do we normally do?

Reply:
Usually we bump deps only when really required. Sometimes even bend over backwards to support older versions. But, realistically, I don't think we need to support dimod from Dec 2018. OTOH, qbsolv hasn't seen much dev work and updates since then, and it costs us nothing to allow 0.8.. :shrug:

Reply:
But qbsolv is specific as it's relatively self-standing (not requiring the whole Ocean), so I vote to support 0.8. Until we find a good excuse to bump.

Reply:
Makes sense to me. Will remove this change!